presto-main/src/test/java/com/facebook/presto/type/TestRowOperators.java

```diff
@@ -190,8 +204,28 @@ public void testRowEquality()
             //Expected
         }
 
+        try {
```

Comment:
This test class is relatively old. I would add a commit before your current one that turns the existing usage of `try` in this class into `assertInvalidFunction`.

Please rewrite your usage of `try` here to it as well.